src/main/java/homework3/Task1.java

```diff
@@ -0,0 +1,18 @@
+package homework3;
+
+public class Task1 {
+    public static void main(String[] args) {
+        int a = 13;
+        float b = 6.55f;
+        float SumResult = a+b;
```

Comment:
variables names should start with lowercase letter
```suggestion
        float sumResult = a + b;
```